src/design/index.md

```diff
@@ -97,21 +103,75 @@ Interaction design along with other disciplines helps us create products that:
 
 ### When to do it
 
-Interaction design is applicable throughout the agile delivery phases: Discovery, Alpha, Beta and Live (as defined by GDS). In addition, Interaction designers also work on upgrading legacy applications and rolling out changes to ensure internal NHS services are consistent and meet current standards and regulations.
+Interaction Design is applicable throughout the agile delivery phases: Discovery, Alpha, Beta and Live (as defined by GDS). In addition, Interaction Designers also work on upgrading legacy applications and rolling out changes to ensure internal NHS services are consistent and meet current standards and regulations.
 
 ### Benefits
 
-Interaction design supports digital delivery to ensure benefits can be realised
+Interaction Design supports digital delivery to ensure benefits can be realised
 
  * Reduces risk, development and maintenance costs by testing assumptions and hypothesis
  * Improves outcomes by facilitating design sessions to ensure a wide range of possibilities are considered
  * Improves performance and reduces the number of user errors
  * Increases user satisfaction and trust in both the service and the NHS
  * Improves efficiency through consistency and familiarity
 
-## Service example
+### Example
 
-| Service Design | Manage a waste removal company |
+| Type of design | Example |
 | --- | --- |
 | Interaction Design | Apply for a waste Carrier, Broker or Dealer permit |
-| Interaction Design | Apply for a standard rules waste Permit |
\ No newline at end of file
+| Interaction Design | Apply for a standard rules waste Permit |
+
+## Content Design
+
+### What is Content Design?
+
+Content Design is answering a user need in the best way for the user to consume it.
+
+It:
+
+* allows someone to find something quickly and easily
+* is not just about writing web copy
+* can take a range of formats
+* should be based on research
+
+### Why do it?
+
+Good content design:
+
+* helps people get the information they need in a way they expect
+* focuses on the users needs by creating content that is simple and easy to understand
+* makes sure content is accessible and inclusive for anyone who needs it
+* keeps content across multiple services consistent by sticking using style guides and standards
```

Comment:
`sticking to style guides...`?

Reply:
![image](https://user-images.githubusercontent.com/45825845/217861933-85fe66fa-09d7-49d4-9f32-008655f595b9.png)